editor.planx.uk/src/@planx/components/Question/Editor.tsx

```diff
@@ -247,7 +152,10 @@ export const Question: React.FC<Props> = (props) => {
               }) as Option
             }
             Editor={OptionEditor}
-            editorExtraProps={{ showValueField: !!formik.values.fn }}
+            editorExtraProps={{
+              showValueField: !!formik.values.fn,
+              showDescriptionField: true,
```

Comment:
A new prop for toggling the description field - question options have it, while checklists do not